kanidm_rlm_python/Dockerfile

```diff
@@ -51,6 +52,7 @@ RUN python3 -m pip install --no-cache-dir --no-warn-script-location /pkg/kanidmr
 RUN rm -rf /pkg/*
 
 USER radiusd
+ENV LD_PRELOAD=/usr/lib64/libpython3.so
```

Comment:
I'm sure there is a reason for this but .... da fuq?

Reply:
Yeah, python changed something about how they link modules

Reply:
Discussed externally, I'm going to submit a fix for freeradius-server to opensuse :)